lib/create-contentful-api.js

```diff
@@ -1,73 +1,73 @@
 import errorHandler from './error-handler'
 import entities from './entities'
 
+const {wrapSpace, wrapSpaceCollection} = entities.space
 
 /**
- * @typedef {ClientAPI} ClientAPI
```

Comment:
I think we can merge this with `ContentfulManagement` Object as it sound like there is a duplicate there